### ISSUE TITLE
[X-CLIP] testing zero-shot

### ISSUE THREAD

**Issue by shinpaul14** [microsoft/VideoX#64]
In testing zero-shot I kept getting 
`UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")`

this warning, and I was wonder if this effects the performance or not.

**Comment by nbl97** [comment]
Thanks for your interest in our work~
I prefer that it does not affect the test performance : )

**Comment by nbl97** [comment]
I'm closing this issue, but pls feel free to ping me if there are further questions.
